Swat/Swat.php

```diff
@@ -209,7 +209,7 @@ private function __construct()
 	 * gettext.
 	 *
 	 * @package   Swat
-	 * @copyright 2005-2006 silverorange
+	 * @copyright 2005-2015 silverorange
```

Comment:
I know this is just updating the date, but why does `Swat.php` have multiple copyright blocks?